lib/main.dart

```diff
@@ -22,11 +22,13 @@ class MyApp extends StatelessWidget {
     routes: <GoRoute>[
       GoRoute(
         path: routePathRootScreen,
-        builder: (_, __) => const LoginScreen(),
+        builder: (BuildContext context, GoRouterState state) =>
+            const LoginScreen(),
```

Comment:
Please check again as this rebase will revert the old code which violate [this comment](https://github.com/nimblehq/flutter-ic-david-bruce/pull/45#discussion_r1162284521)